src/sctools/test/test_metrics.py

```diff
@@ -344,7 +344,7 @@ def test_higher_order_metrics_by_gene(metrics, key, expected_value):
     the test suite across all measured instances of the metric class. E.g. for cell metrics (class),
     each test will verify the value for each cell (instance).
 
-    :param pd.DataFrame metrics: Output from subclass of sctools.metrics.SequenceMetricAggregator
+    :param pd.DataFrame metrics: Output from subclass of sctools.metrics.MetricAggregator
```

Comment:
It seems this test file is still using reST style?

Reply:
I figure test cases should have minimal doc strings to save the time of developers. So I'm happy with it if you don't apply numpy style on these files.